docs/design/byos.md

```diff
@@ -66,78 +75,39 @@ Not offered:
+•	Customer can pick a [size](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes?toc=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fazure%2Fvirtual-machine-scale-sets%2FTOC.json&bc=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fazure%2Fbread%2Ftoc.json) for their virtual machines.
+•	Customer can use [availability zones](https://docs.microsoft.com/en-us/azure/availability-zones/az-overview) to distribute VM instances in a scale set across multiple data centers.
+•	Customer can configure [networking](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-networking) for VMs in their scale set.
+•	Customer can schedule automatic [OS image updates](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-automatic-upgrade).
+•	Customer can use standard Azure policies to control various settings in the scale set.
+•	Azure Pipelines can easily grow or shrink the number of virtual machines in the scale set.
 
-### Custom / configurable SKU
+Azure has a good [marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps?filters=virtual-machine-images) of images that customers can start with for their base images. These include Windows Server, Ubuntu, Debian based images, Windows server with Visual Studio, etc.
+In addition, we will publish our Microsoft-hosted images to this marketplace.
 
-Offering additional Azure SKUs in Hosted will meet some of the above scenarios.
-Specifically, if all you need are additional resources (memory, compute, etc.), then a custom SKU works.
-
-Pros:
-- Easiest for the customer to understand
-- Potential business model around premium SKUs
-
-Cons:
-- Capacity planning and buildout becomes a lot more complex
-- If a customer wants a SKU we don't offer, they're out of luck until we add it
-- Does not offer custom elasticity, network environment, or image
-
-### Bring your own cloud (pool providers)
-
-Putting the entire agent alloc/release cycle in the hands of the customer offers them ultimate flexibility.
-
-Pros:
-- Maximum flexibility
-- Customer can select other infrastructure: on-prem, in a different hosting provider, etc.
-
-Cons:
-- Much heavier burden to understand, set up, and maintain
-- Customer has to swallow full complexity load to get any flexibility
+### Setup
 
-<!--
-older section
-## Goals
+For a lot of customers, it would be enough to have them
+- go to pool setup
+- create a new pool, and select the option to use Azure VMs
+- pick an Azure subscription, a VM scale set + a few different agent lifetime policies
+- have Azure Pipelines configure it all for them
 
```

Comment:
What is the scope of the token that Azure Dev Ops provisions for itself when managing VMs in a user's subscription. We would want to do everything possible to limit contagion i.e. an exploit in one users subscription making it to another user's subscription and agents